### PR TITLE
Update SPEC.md

### PR DESCRIPTION
**Pull request by zqb-all**
fix size in Layout of the CRC tag

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -764,9 +764,9 @@ Layout of the CRC tag:
 ```
         tag                                    data
 [--      32      --][--      32      --|---        variable length        ---]
-[1| 3| 8 | 10 | 10 ][--      32      --|---            (size)             ---]
+[1| 3| 8 | 10 | 10 ][--      32      --|---          (size - 4)           ---]
```
Comment (geky) [inline SPEC.md]: Should this actually be `(size - 32)` since the other fields are in bits?

File: SPEC.md [hunk 2]
```diff
@@ -764,9 +764,9 @@ Layout of the CRC tag:
...
  ^  ^  ^    ^    ^            ^- crc                      ^- padding
- |  |  |    |    '- size (12)
```
Comment (geky) [inline SPEC.md]: What does this 12 mean here? The size could be any value.

File: SPEC.md (hunk 1 above)
Comment (zqb-all) [reply SPEC.md]: yes, this field should in bits, but size is in bytes, maybe `(size * 8 - 32)` is better ?

Comment (geky) [reply SPEC.md]: Ah! Good point, you're right that is better.

**Comment by geky** [comment]
Hi @zqb-all, sorry about the absence. Does the updated comment make sense to you?

If you're not able to update the PR I can bring it in and make the suggested changes on the next release.

**Comment by zqb-all** [comment]
Hi @geky, thanks for you reply, I am glad to update the PR.

**Comment by geky** [comment]
Thanks for the contribution!

**Pull request merged**